uber/templates/registration/new.html

```diff
@@ -168,18 +171,19 @@
             {% if c.NUMBERED_BADGES %}
                 <td>
                     &nbsp;&nbsp;&nbsp;&nbsp;
-                    <input type="number" name="badge_num" size="5" style="width:5em" />
+                    <input type="number" name="badge_num" size="5" style="width:5em" {% if attendee.paid == c.NOT_PAID %}disabled{% endif %} />
                 </td>
             {% endif %}
             <td>
                 {% csrf_token %}
                 <input type="hidden" name="id" value="{{ attendee.id }}" />
-                <input type="submit" value="Check In" {% if attendee.paid == c.NOT_PAID %}disabled{% endif %} />
+                <button class="btn btn-primary" {% if attendee.paid == c.NOT_PAID %}disabled{% endif %}>Check In</button>
```

Comment:
While I was in this file anyway, I changed this button to use bootstrap styling, which also makes the disabled button look disabled.